packages/salesforcedx-vscode-core/src/commands/functions/functionService.ts

```diff
@@ -52,6 +52,10 @@ export interface FunctionExecution extends Terminable {
    * Active debug session attached
    */
   debugSession?: vscode.DebugSession;
+  /**
+   * Flag to determine whether running in a container
```

Comment:
Nit pick - Can we add a "why" here. i.e. we need this flag in invoke/debug which is a separate operation and we won't have context as to which function was started.

Reply:
Never mind. Let's keep it this way. I think the "why" part is quite obvious.